hngi/node-url-shortener
# Chore setup test

### What does this PR do?

* Setup test with mocha and chai

### Description of Task to be completed?

* install express (since it wasn't installed)
* install mocha and chai assertion library
* create index route and 404 routes
* create test index.js file

### How should this be manually tested?

* Check CONTRIBUTING.MD file to setup local repository
* Go to 
	`http://localhost:3000/api/v1/` for index route
	`http://localhost:3000/api/v1/any_url` for page not found route

### Any background context you want to provide?
I had to setup express(not my task) the src/index.js file can be overwritten when the original task owner pushes their code.
